mudita/mudita-center
# [CP-603] change ftp to s3

Jira: [CP-604]

**Description**
Implementing S3 contcept

Testing path:
- run `sm-cli` fetch command to get lattes envs
- change NODE_ENV to production
- trigger the crash-dump creation [OS build](https://drive.google.com/drive/u/3/folders/13lRoWZWEQLGI-Mv1HflTTmeMlZnWsJcb)
- send the crash dump
- log in to AWS and open the S3 bucket, you should see the upload crash dump file the timestamp

<details>
<summary><b>Screenshots</b></summary>
// put images here
<img width="1503" alt="Screen Shot 2021-11-09 at 2 33 35 PM" src="https://user-images.githubusercontent.com/21984930/140934607-b19ad77b-1e08-45a0-9c0d-81f63e2fdca5.png">


</details>


[CP-604]: https://appnroll.atlassian.net/browse/CP-604?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
